tenet/chain.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "tenet",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://tenet.org/",
+  "pretty_name": "Tenet",
+  "chain_id": "tenet_1559-1",
+  "bech32_prefix": "tenet",
+  "node_home": "$HOME/.tenetd",
+  "daemon_name": "tenetd",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "atenet",
+        "fixed_min_gas_price": 250000000,
+        "low_gas_price": 20000000000,
+        "average_gas_price": 25000000000,
+        "high_gas_price": 40000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "atenet"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/tenet-org/tenet-mainnet",
+    "recommended_version": "v11.0.6",
+    "compatible_versions": [
+      "v11.0.6"
+    ],
+    "binaries": {
+      "linux/amd64": "https://github.com/tenet-org/tenet-mainnet/releases/download/v11.0.6/tenet-mainnet_11.0.6_Linux_amd64.tar.gz",
+      "linux/arm64": "https://github.com/tenet-org/tenet-mainnet/releases/download/v11.0.6/tenet-mainnet_11.0.6_Darwin_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/tenet-org/tenet-mainnet/releases/download/v11.0.6/tenet-mainnet_11.0.6_Darwin_amd64.tar.gz",
+      "darwin/arm64": "https://github.com/tenet-org/tenet-mainnet/releases/download/v11.0.6/tenet-mainnet_11.0.6_Darwin_arm64.tar.gz",
+      "windows/amd64": "https://github.com/tenet-org/tenet-mainnet/releases/download/v11.0.6/tenet-mainnet_11.0.6_Windows_amd64.zip"
+    },
+    "cosmos_sdk_version": "0.46",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "ibc_go_version": "6.1.0",
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/tenet-org/tenet-mainnet/master/config/genesis.json"
+    },
+    "versions": []
```

Comment:
Can you please also copy the version data you provided above into an object in this versions array?